packages/compass-connections/src/components/connection-list/connection-list.tsx

```diff
@@ -180,6 +210,13 @@ function ConnectionList({
           >
             Saved connections
           </H3>
+          <ItemActionControls<FavoriteAction>
```

Comment:
sgtm! done in e1a4f7cc1805ac90e2b7e3716946e1cb716dab03

Now that I’m thinking about it, I’m not sure if the idea really was to only show the menu opener button on hover… it would feel pretty inconsistent to me to not follow what we do for the rest of the connections list sidebar, though

Reply:
Yeah, we do the same for "Clear all" button that shows up in "Recents" list, so even for list headers the pattern seems to be there. For a feature like this it's nice to expose it in more accessible places I think, so thanks for adding this!